sklearn/cluster/k_means_.py

```diff
@@ -874,6 +874,9 @@ def fit(self, X, y=None):
         Parameters
         ----------
         X : array-like or sparse matrix, shape=(n_samples, n_features)
+            Numerical matrix of training samples to cluster. Each row
+            will represent a single training sample, a point in space of
+            'n_features' dimensions.
```

Comment:
This is the first draft of description. Commenting here so that this can be seen directly in PR conversation even after this diff gets outdated by any modifications.